### PR TITLE
add script to generate geth genesis with preloaded contract accounts

### PR DESCRIPTION
**Pull request by jwasinger**

### CODE REVIEW

File: genesis.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+for arg in args.precompiles:
+  if len(args.split(",")) != 2:
+    raise Exception("invalid argument for precompiles {0}".format(args))
+
+  address = arg.split(",")[0] #TODO validation
+  codefile = arg.split(",")[1]
+
+  try:
+    with open(codefile, 'rb') as f:
+      genesis['alloc'][address] = binascii.hexlify(f.read())
```
Comment (axic) [inline genesis.py]: This should also have the option to set initial balance.

Also would be nice to actually find and update an account entry if it exists.

Comment (jwasinger) [reply genesis.py]: Done.

**Comment by jwasinger** [comment]
Added geth genesis.  Changed preloaded accounts argument format to one json object, e.g. 

`python genesis.py --genesis geth-genesis.json --preload '{"0x00":{ "balance": "0x00", "code": "0xffff"}}'`

**Comment by axic** [comment]
> Changed preloaded accounts argument format to one json object, e.g.

Does it mean that one would need to paste in 500k of hexstring on the CLI?

**Comment by jwasinger** [comment]
No.  because the accounts take `codefile` as an argument.

**Comment by axic** [comment]
@jwasinger what is the status of this?

**Comment by axic** [comment]
@jwasinger what is the status of this?

**Comment by jwasinger** [comment]
Irrelevant now.

**Comment by jwasinger** [comment]
Let me save the branch

**Pull request closed without merging**